.prettierrc.yaml

```diff
@@ -1,7 +1,8 @@
+arrowParens: avoid
```

Comment:
Now there is an options if we the parenthesis back :)

Reply:
I definitely want them

Reply:
The we have to take a vote I guess :)

https://vote4code.com/5764748591235072/arrow-function-parentheses#red